src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -436,7 +436,7 @@ public final boolean onToolClick(EntityPlayer playerIn, @Nonnull Set<String> too
             } else return onScrewdriverClick(playerIn, hand, gridSideHit, hitResult);
         }
         if (toolClasses.contains(ToolClasses.SOFT_MALLET)) {
-            if (coverBehavior != null && coverBehavior.onSoftMalletClick(playerIn, hand, hitResult) == EnumActionResult.SUCCESS) {
+            if (getCoverAtSide(hitResult.sideHit) != null && getCoverAtSide(hitResult.sideHit).onSoftMalletClick(playerIn, hand, hitResult) == EnumActionResult.SUCCESS) {
```

Comment:
Originally, this logic was for checking the entire side of a meta tile entity instead of tracing with the machine grid. After discussion, it has been removed in https://github.com/GregTechCEu/GregTech/pull/1414/commits/b78be251c1d22fd89fc37d9436a154bb724d7c36.